### ISSUE TITLE
 would like to know if there would be a possibility to use the climate indices library, but my data are in accumulated quarterly precipitation.  For example, I have my JJA file from 1990 to 2019.

### ISSUE THREAD

**Issue by ValescaFernandes** [monocongo/climate_indices#372]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when \[...]

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by ValescaFernandes** [comment]
I would like to know if there would be a possibility to use the climate indices library, but my data are in accumulated quarterly precipitation.

For example, I have my JJA file from 1990 to 2019.

**Comment by monocongo** [comment]
Yes, this should be possible. We would need to add a new periodicity type with 4 periods per year to account for this type of granularity.

**Comment by ValescaFernandes** [comment]
Ok, and could the script be made and updated with this 3 month change?

Thanks

**Comment by monocongo** [comment]
Yes, I think so, although I can't promise that I can have it done for you anytime soon as I currently have very limited bandwidth for it. If you want to give it a whirl and submit a PR I will be happy to review/consider it.

Essentially you can think of it like this: for monthly data we're binning the data into 12 bins (one per month) per year and for daily data we're binning the data into 366 bins (one per day), so for your use case we'll be binning the data into four bins per year. Other than this everything else should more or less work the same so I don't see this as too big of a lift.

**Comment by monocongo** [comment]
Hi @ValescaFernandes can you please provide a sample quarterly precipitation dataset that I can use as an example input? This will facilitate development of this new functionality.

**Comment by bennyistanto** [comment]
I think this is very good suggestion for enhancement.
And quarterly precipitation is not just 4periods per year, but 12. Probably rolling accumulation for each period is better than only 4periods per year: DJF, JFM, FMA, MAM, AMJ, MJJ, JJA, JAS, ASO, SON, OND, NDJ

This is also useful for daily precipitation data. For example: to make monitoring (SPI) more frequently by calculate SPI1 to SPIx and update every dekad.

I could provide example data for 1,3,6,9,12 and 24month rainfall rolling accumulation by dekad.
